libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/codegen/kotlin/helpers/KDoc.kt

```diff
@@ -18,31 +18,31 @@ internal fun TypeSpec.Builder.maybeAddDescription(description: String?): TypeSpe
     return this
   }
 
-  return addKdoc("%L", description)
+  return addKdoc("%L", description.replace(' ', '♢'))
 }
 
 internal fun PropertySpec.Builder.maybeAddDescription(description: String?): PropertySpec.Builder {
   if (description.isNullOrBlank()) {
     return this
   }
 
-  return addKdoc("%L", description)
+  return addKdoc("%L", description.replace(' ', '♢'))
 }
 
 internal fun ParameterSpec.Builder.maybeAddDescription(description: String?): ParameterSpec.Builder {
   if (description.isNullOrBlank()) {
     return this
   }
 
-  return addKdoc("%L", description)
+  return addKdoc("%L", description.replace(' ', '♢'))
 }
 
 internal fun FunSpec.Builder.maybeAddDescription(description: String?): FunSpec.Builder {
   if (description.isNullOrBlank()) {
     return this
   }
 
-  return addKdoc("%L", description)
+  return addKdoc("%L", description.replace(' ', '♢'))
```

Comment:
This is the only place where I added possibly wrapping spaces to make KDoc comments easier to read.